### PR TITLE
[Japneet | Procheta] Add. select api workflow

### PR DESCRIPTION
**Pull request by japneetkaurbhatia**

### CODE REVIEW

File: local.env [hunk 1]
```diff
@@ -1,4 +1,5 @@
 BUYER_APP_PORT=2010
 BUYER_APP_ID=sample_mobility_bap
 BUYER_APP_URL=http://mobility-sample-bap:2010
-GATEWAY_URL=http://sample-gateway-registry:1010
\ No newline at end of file
+GATEWAY_URL=http://sample-gateway-registry:1010
+SELLER_APP_URL=http://mobility-sample-bpp:3010
```
Comment (hariharanweb) [inline local.env]: you cant add BPP to process. It should be in context

Comment (hariharanweb) [reply local.env]: Also run `npm run lint`

Comment (hariharanweb) [reply local.env:5]: Test

Comment (japneetkaurbhatia) [reply local.env]: Updated in [frontend]

(https://github.com/hariharanweb/mobility-sample-frontend/pull/13/commits/e6871b9f9dfd16d7a27eb5d49edb99bca938b2f0)

and in [BAP]
(https://github.com/hariharanweb/mobility-sample-bap/pull/1/commits/dca23247f7c70ef38e0b48b1a1a3cd0aab5a9b50)

**Pull request merged**